purchase_procurement_analytic/__manifest__.py

```diff
@@ -16,6 +16,7 @@
     'depends': [
         'purchase',
         'procurement_analytic',
+        'stock_analytic',
```

Comment:
I don't understand why you need this new dependency

Reply:
@rousseldenis me too.
I do tests without stock_analytic dependency and module works fine. Why this dependency?